CHANGELOG.md

```diff
@@ -1,3 +1,36 @@
+## Atomic App 0.5.2 (05-24-2016)
+
+This release of Atomic App we include binary generation, a logging refactor/clean-up as well as a minor code refactor to the main Nulecule codebase.
+
+The main features are:
+  - Binary generation
```

Comment:
Could use some more context. - `Add support for generating a static binary of Atomic App.` ??